modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/components/ListView/ListView.tsx

```diff
@@ -178,6 +182,18 @@ const ListViewRest: React.FC<ListViewProps> = ({
 		}
 	}, [listViewContext, onContextChange]);
 
+	useEffect(() => {
+		if (tableProps.rowSelectable) {
+			const initialChecked = items.every(({id}) =>
+				selectedRows.includes(id)
+			);
+			dispatch({
+				payload: initialChecked,
+				type: ListViewTypes.SET_CHECKED_ALL_ROWS,
+			});
+		}
+	}, [items, tableProps, selectedRows, dispatch]);
```

Comment:
Add space

```suggestion
	useEffect(() => {
		if (tableProps.rowSelectable) {
			const initialChecked = items.every(({id}) =>
				selectedRows.includes(id)
			);
			
			dispatch({
				payload: initialChecked,
				type: ListViewTypes.SET_CHECKED_ALL_ROWS,
			});
		}
	}, [items, tableProps, selectedRows, dispatch]);
```

or inline

```suggestion
	useEffect(() => {
		if (tableProps.rowSelectable) {
			dispatch({
				payload: items.every(({id}) =>
				     selectedRows.includes(id)
			        ),
				type: ListViewTypes.SET_CHECKED_ALL_ROWS,
			});
		}
	}, [items, tableProps, selectedRows, dispatch]);
```